### DISCUSSION TITLE
Julia lessons!

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#307] [Ideas]
Interested in contributing Julia lessons? Let's go!

**Comment by ayush714** [comment]
@jlooper Yaa, I am interested, Should I start with the first assignment solutions?

**Reply by jlooper** [reply]
yes, sounds great. We have been putting the R lessons in the 'Solutions' folder, but if we add Julia, do you think we should create a new folder structure? I don't mind adding 'R' and 'Julia' folders within the 'solution' folders

**Reply by ayush714** [reply]
Here are the things we can do:- 
1.) Creating a separate folder for Julia solutions:- 
It will be good we will be having separate folders for each assignment solutions, so in future, I we wanna to add more languages then that would not clutter up things.  So we should go for it.

**Comment by jlooper** [comment]
@ayush714 do you plan to continue this work or should I close this conversation? We can only accept ports of our lessons into new languages (like Eric did with R).

**Comment by ayush714** [comment]
Hey, I am doing. I am planning to put it all at once, I am working, don't
close this.

**Reply by sudhansubhushanmishra** [reply]
Could it please be possible to be involved in this venture of porting exercises to the Julia language?

**Comment by Bassamejlaoui** [comment]
I am Interested to contribute

**Comment by sudhansubhushanmishra** [comment]
Is this still open to contributions? I'd love to contribute.
